### ISSUE TITLE
Error when use command `:Femaco`

### ISSUE THREAD

**Issue by tan-wei** [AckslD/nvim-FeMaco.lua#32]
I use default configure with setup, but will cause error:
```
Error executing Lua callback: /usr/share/nvim/runtime/lua/vim/treesitter.lua:189: attempt to index local 'node_or_range' (a nil value)
stack traceback:
	/usr/share/nvim/runtime/lua/vim/treesitter.lua:189: in function 'get_match_range'
	...ocal/share/nvim/lazy/nvim-FeMaco.lua/lua/femaco/edit.lua:84: in function 'get_match_at_cursor'
	...ocal/share/nvim/lazy/nvim-FeMaco.lua/lua/femaco/edit.lua:156: in function 'edit_code_block'
	...ocal/share/nvim/lazy/nvim-FeMaco.lua/lua/femaco/init.lua:12: in function <...ocal/share/nvim/lazy/nvim-FeMaco.lua/lua/femaco/init.lua:12>
```

I can confirm:
  - Neovim is the latest nightly version
  - All treesitters are updated

**Comment by AckslD** [comment]
Hmm, seems indeed like the tables returned by treesitter has changed

**Comment by tan-wei** [comment]
It always happens. 🤣

**Comment by tan-wei** [comment]
Still met the problem with version `v0.10.0-dev-948+g0b0d912763`.
I want to try the latest Neovim version (`v0.10.0-dev-962+g874b8172a6`), but Neovim will crash on startup, so I could only use an older one.



Update:

After disable some plugins. I can use `v0.10.0-dev-962+g874b8172a6`
But still:
```
Error executing Lua callback: /usr/share/nvim/runtime/lua/vim/treesitter.lua:189: attempt to index local 'node_or_range' (a nil value)
stack traceback:
	/usr/share/nvim/runtime/lua/vim/treesitter.lua:189: in function 'get_match_range'
	...ocal/share/nvim/lazy/nvim-FeMaco.lua/lua/femaco/edit.lua:42: in function 'get_match_text'
	...ocal/share/nvim/lazy/nvim-FeMaco.lua/lua/femaco/edit.lua:59: in function 'parse_match'
	...ocal/share/nvim/lazy/nvim-FeMaco.lua/lua/femaco/edit.lua:91: in function 'get_match_at_cursor'
	...ocal/share/nvim/lazy/nvim-FeMaco.lua/lua/femaco/edit.lua:164: in function 'edit_code_block'
	...ocal/share/nvim/lazy/nvim-FeMaco.lua/lua/femaco/init.lua:12: in function <...ocal/share/nvim/lazy/nvim-FeMaco.lua/lua/femaco/init.lua:12>
```

**Comment by AckslD** [comment]
@tan-wei I think it should now finally be fixed by #35 :crossed_fingers:

**Comment by tan-wei** [comment]
Yeah! I can confirm. Great!
